### ISSUE TITLE
speed up tilt_map: parallel apply

### ISSUE THREAD

**Issue by marcosci** [marcosci/layer#3]
tilt_map is currently reasonably fast, but applied to large raster ( > 5000x5000 cells) it is kind of a bottleneck.

We could simply wrap the last line of the function where we apply the shear and rotate matrix in a future_map or similar.

Will be implemented, as soon as someone needs this.

**Comment by marcosci** [comment]
Big fan of dependency creep, if they are packages like you mentioned :) Happy for your PR Joshua!

**Comment by marcosci** [comment]
Hi Joshua!

I merged your solution and it looks very promising, thanks a ton!

Here are the results for the US dataset with wildfires:

![image](https://user-images.githubusercontent.com/10864574/148754326-7f26181c-5346-49d3-8c0c-918efcd81297.png)

Raster is 800 x 350 cells. I added you as contributor, hope that is fine! Couldn't find your mail, if you want it to be included ... that would be another PR or you send it to me via DM on twitter :)

Cheers
